packages/block-library/src/query/edit/inspector-controls/order-control.js

```diff
@@ -27,6 +27,7 @@ const orderOptions = [
 function OrderControl( { order, orderBy, onChange } ) {
 	return (
 		<SelectControl
+			__nextHasNoMarginBottom
```

Comment:
## Testing instructions

In the editor, add a Query Loop block and see the block inspector. Toggle the "Query type" to "Custom".

<img src="https://github.com/user-attachments/assets/124ebeba-f27e-435a-b1b5-fa912dd1f0d2" alt="Query Loop block inspector" width="273">